CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.h

```diff
@@ -1,6 +1,12 @@
 #ifndef ECALTPGPARAMBUILDER_H
 #define ECALTPGPARAMBUILDER_H
 
+//Author: Pascal Paganini - LLR
+//Date: 2006/07/10 15:58:06 $
+
+#define CMSSW_VERSION 340
```

Comment:
Sorry, compilation does NOT work any more, removing this test.  Many references for EcalShape in .cc...I don't know if it is used elsewhere.